PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -769,6 +769,12 @@ generic_string stringToUpper(generic_string strToConvert)
     return strToConvert;
 }
 
+generic_string stringToLower(generic_string strToConvert)
+{
+    std::transform(strToConvert.begin(), strToConvert.end(), strToConvert.begin(), ::tolower);

```

Comment:
AFAIK the project is build with both `UNICODE` and `_UNICODE` defined.
As `generic_string` is defined as `std::basic_string<TCHAR>`, will make it a wide character string.
So shouldn't it be `::towlower` ?

Reply:
Just to be on the safe side, always cast the argument to `::tolower` to `unsigned char`, and likewise the argument to `::towlower` to `wchar_t`. Otherwise the behavior is undefined. (ISO C99, 7.25.1p5)

Reply:
done